src/components/dev-hub/project-sign-up-form.js

```diff
@@ -73,18 +73,22 @@ const Form = React.memo(({ setSuccess, success }) => {
             </ErrorMessage>
             <StyledInput
                 value={name}
+                maxLength="50"
                 required
                 placeholder="Name"
+                pattern="^[A-Za-zÀ-ÿ ,.'-]+$"
```

Comment:
This is great! The only issue is with UX for the error message. [Check out this article](https://webdesign.tutsplus.com/tutorials/html5-form-validation-with-the-pattern-attribute--cms-25145), and can we add a more user-friendly error message?